scripts/install/functions.sh

```diff
@@ -65,7 +65,6 @@ function prepare_install {
     ng config -g cli.packageManager npm
 
     mkdir -p ${INSTALLATION_DIR}
-    ng analytics off
```

Comment:
to the reviewer:

line 343 of this file has ng build. Reason for that it is the 'customers' application and not our development one. Basically, we're converting source development to use Nx, but customers is business as usual.

Reply:
What is the purpose of ng analytics off?